test/fortran/test_fortran_spg_get_symmetry.f90

```diff
@@ -1,49 +1,98 @@
-function test_fortran_spg_get_symmetry() bind(C) result(ret)
+program test_fortran_spg_get_symmetry
     use, intrinsic :: iso_c_binding
     use spglib_f08, only: spg_get_symmetry
     use test_utils
     implicit none
 
-    real(c_double) :: lattice(3, 3)
-    real(c_double) :: position(3, 12)
-    integer(c_int) :: types(12)
+    write (*, '("test_rutile112")')
+    call test_rutile112()
+    write (*, '("test_zincblende")')
+    call test_zincblende()
```

Comment:
Note that using such bundling and a `stop`, failing one test will fail them all.

But also, this can be achieved with the function approach. There we can have as many function/subroutines we want in the file, as long as there is at least one function with the appropriate name and input/output format.

Reply:
I understand this is a dirty approach. I could not find a better solution.